charts/tidb-cluster/templates/monitor.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.monitor.create }}
+apiVersion: pingcap.com/v1alpha1
+kind: TidbMonitor
+metadata:
+  name: {{ template "cluster.name" . }}-monitor
+  namespace: {{ .Release.Namespace }}
```

Comment:
Please add the label information like other components defined in helm chart template.